Lib/test/test_zipfile.py

```diff
@@ -1718,6 +1719,14 @@ def test_non_existent_file_raises_OSError(self):
         # quickly.
         self.assertRaises(OSError, zipfile.ZipFile, TESTFN)
 
+    def test_issue44439(self):
```

Comment:
It would be better to move this test to AbstractWriterTests and test with different compressions.